### ISSUE TITLE
It just Doesn't work

### ISSUE THREAD

**Issue by polyinnovator** [Quorafind/Obsidian-Big-Calendar#41]
I have tried this plugin on multiple occasions over the last couple years, and it just seems to not want to work.

It SAYS successfully opened, but then it is just a blank screen. I even did a side install via github download instead of the plugin directory. I've done both more than a few times.

Was wondering if anyone else has this issue, and if the dev is still working

**Comment by alibabarahaei** [comment]
It looks like, when you wait after a while or restart app It will be fixed by itself.

**Comment by alibabarahaei** [comment]
> I have tried this plugin on multiple occasions over the last couple years, and it just seems to not want to work.
> 
> It SAYS successfully opened, but then it is just a blank screen. I even did a side install via github download instead of the plugin directory. I've done both more than a few times.
> 
> Was wondering if anyone else has this issue, and if the dev is still working

It looks like, when you wait after a while or restart app It will be fixed by itself.

**Comment by polyinnovator** [comment]
Yeah that's why I commented on here a while back.. It just doesn't work anymore. When I first tried it I was able to actually see a cal., but that was a while back. Since then its never done anything no matter how you install.

**Comment by incantatem1** [comment]
doesn't work for me either. maybe it's not compatible with newer versions of obsidian anymore?
